BAM/src/main/java/com/revature/bam/logging/ServiceAndControllerLogger.java

```diff
@@ -176,10 +184,10 @@ public Object classNameAndTime(ProceedingJoinPoint pjp) throws Throwable {
 	 * @param getMapping
 	 */
 	@Pointcut("@target(classRequestMapping) && @annotation(getMapping) && execution(* com.revature.bam.controller.*.*(..))")
-	public void controller(RequestMapping classRequestMapping, GetMapping getMapping) {}
+	public void	controller(RequestMapping classRequestMapping, GetMapping getMapping) {}
```

Comment:
Is there meant to be an empty method here?

Reply:
Yes, it's necessary to make a pointcut for annotations.